bootstrap/kubeadm/controllers/kubeadmconfig_controller.go

```diff
@@ -107,6 +107,7 @@ func (r *KubeadmConfigReconciler) SetupWithManager(mgr ctrl.Manager, option cont
 				ToRequests: handler.ToRequestsFunc(r.ClusterToKubeadmConfigs),
 			},
 		).
+		WithEventFilter(util.external.FilterPausedAnnotations).
```

Comment:
```suggestion
		WithEventFilter(external.FilterPausedAnnotations).
```

Autocomplete strikes back? :P